### DISCUSSION TITLE
How does code promotion to NuGet work?

### DISCUSSION THREAD

**Discussion by DSPLX** [elsa-workflows/elsa-core#3320] [Ideas]
Could the CONTRIBUTING.md file be updated to describe how and when we can expect new features from PRs to be available on NuGet? I'm not exactly sure how to find the latest changes as they don't appear to be included in 2.9.0.

**Comment by johnwc** [comment]
Also, we're seeing 2.9.0 available in nuget but not seeing it officially released in github repo. Is what is in nuget, full release or preview? The current released code in nuget fails for any DB other than sqlite.

**Reply by sfmskywalker** [reply]
@johnwc I have yet to create the release notes. What is on NuGet is indeed the full release, not the preview.
I'll take a look at the DB issue soon.

**Comment by sfmskywalker** [comment]
@DSPLX That's a good idea.
